internal/builders/docker/actions/create-predicate/src/main.ts

```diff
@@ -0,0 +1,94 @@
+import * as core from "@actions/core";
```

Comment:
Do you think we should move this action under the `.github/actions`? I think they are internal too. If we add tests for all actions (e.g., https://github.com/slsa-framework/slsa-github-generator/issues/1513), it would avoid missing this action.